.github/workflows/test_pull_requests.yml

```
@@ -201,10 +208,13 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
+        with:
+          path: napari-from-github
+
       - uses: actions/setup-python@v4
         with:
-          python-version: 3.9
-          cache-dependency-path: setup.cfg
+          python-version: 3.11
```

As we run here `tox -e py39-linux-pyside2-examples`, then python 3.9 need to be installed or python version for check examples need to be changed

---

Should we rename to "test examples are working on older supported version" ?
```suggestion
          python-version: 3.9
```

And do we want to test examples on the newest python version ?

---

It was tested working on the middle version when implemented (as expected as more popular). So maybe it should be moved to python 3.10.